### ISSUE TITLE
Fix when solver doesn't support objective

### ISSUE THREAD

**Issue by odow** [jump-dev/MultiObjectiveAlgorithms.jl#61]
This example fails because HiGHS doesn't support `VariableIndex` objectives
```julia
using JuMP
import MultiObjectiveAlgorithms as MOA
import HiGHS
model = Model(() -> MOA.Optimizer(HiGHS.Optimizer))
set_silent(model)
set_attribute(model, MOA.Algorithm(), MOA.KirlikSayin())
@variable(model, x >= 0, Int)
@variable(model, y >= 0, Int)
@constraint(model, x + y >= 1)
@objective(model, Min, [x, y])
optimize!(model)
```

**Comment by mzy2240** [comment]
Might be a dumb question but do we support syntax like this:
```julia
@objective(model, Min, x)
@objective(model, Min, y)
optimize!(model)
```

**Comment by odow** [comment]
> Might be a dumb question but do we support syntax like this?

No. Calling `@objective` a second time replaces the current objective.

**Comment by odow** [comment]
Closed by #63
